rg_instructor_analytics/static/rg_instructor_analytics/js/EnrollmentTab.js

```diff
@@ -106,11 +107,11 @@ function EnrollmentTab(button, content) {
             var layout = {
                 hovermode:'closest',
                 xaxis: {},
-                yaxis: {dtick: 1},
-                showlegend: false,
+                yaxis: {nticks: 4},
+                showlegend: false
             };
             var data = [unenrollTrace, enrollTrace, totalTrace];
-
+            console.log(data);
```

Comment:
Please remove this log message or provide a more suitable description.

Reply:
done